events/views.py

```diff
@@ -25,21 +25,26 @@ def index(request):
     oldEvents = Event.objects.filter(
         active=True, eventDate__lt=now).order_by('-eventDate')
 
+    categories = Category.objects.all()
+
+    formFilter = FilterEventSearch()
```

Comment:
@sebasmagri  I use camel case because the project uses this. Do we make the changes ?. The models also use camelcase.